modules/installing-gcp-user-defined-labels-and-tags.adoc

```diff
@@ -0,0 +1,148 @@
+
+.Sample `infrastructure` file
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Infrastructure
+metadata:
+ name: cluster
+spec:
+ platformSpec:
+   type: GCP
+status:
+ infrastructureName: <cluster_id>
+ platform: GCP
+ platformStatus:
+   gcp:
+     resourceLabels:
+     - key: <label_key>
+       value: <label_value>
+     resourceTags:
+     - key: <tag_key_short_name>
+       parentID: <OrganizationID/ProjectID>
+       value: <tag_value_short_name>
+   type: GCP
+----
+
+Along with the user-defined labels, resources will also have a label defined by the {product-title}. The format of the {product-title} labels is `kubernetes-io-cluster-<cluster_id>:owned`. The `cluster_id` parameter is the value of `.status.infrastructureName` present in `infrastructures.config.openshift.io/cluster`.
+
+User-defined and {product-title} labels can be found on the following GCP resoucres:
+
+* Compute Instance
+* Compute Disk
+* Compute Forwarding Rule
+* Compute Image
+* DNS Managed Zone
+* Filestore Instance
+* Storage Bucket
+
+User-defined tags can be found added to the GCP resources, that is, storage bucket.
+
+The following are the limitations for the user-defined tags:
+
+* Tags will not be attached to master compute instances and storage bucket created by `installer`, worker compute instances created by `machine-api-provider-gcp controller`, filestore instance resources created by `gcp-filestore-csi-driver-operator` and compute disk, compute image resources created by `gcp-pd-csi-driver-operator`.
+* Tags are not supported for buckets located in the following regions:
+** `us-east2` 
+** `us-east3`
+* `cluster-image-registry-operator` will not throw any error but skips processing tags when the buckets are created in the tags unsupported region.
+* Tags must not be restricted to particular service accounts, since operators create and use service accounts with minimal roles.
+* {product-title} will not create tag keys and value resources.
+* {product-title} specific tags will not be added to any resource.
+* Labels for `Compute Address` are supported in the GCP beta version and {product-title} does not add labels to the resource.
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about identifying the `OrganizationID` or `ProjectID`, see:
+** link:https://cloud.google.com/resource-manager/docs/creating-managing-organization#retrieving_your_organization_id[OrganizationID]
+** link:https://cloud.google.com/resource-manager/docs/creating-managing-projects#identifying_projects[ProjectID]
+
+* For more information about tags and labels, see link:https://cloud.google.com/storage/docs/tags-and-labels#tags[Tags and labels].
```

Comment:
We would need different references for tags and labels. Please include below links
Tags: https://cloud.google.com/resource-manager/docs/tags/tags-overview
Labels: https://cloud.google.com/resource-manager/docs/creating-managing-labels

Existing link https://cloud.google.com/storage/docs/tags-and-labels#tags is specific to storage buckets, it won't be required.

Reply:
Ack.